.github/dependabot.yml

```diff
@@ -35,4 +35,17 @@ updates:
         - "mysql_async"
     google-cloud:
       patterns:
-        - "google-cloud*"
\ No newline at end of file
+        - "google-cloud*"
+# Don't update these directories
+- package-ecosystem: cargo
+  directory: /src/workspace-hack
+  schedule:
+    interval: "daily"
+  ignore:
+    - dependency-name: "*"
+- package-ecosystem: cargo
+  directory: /integration_tests/feature-store
```

Comment:
Does it work for subdirectories? `/integration_tests` can meh?

Reply:
idk

Reply:
No we get the answer: no

https://github.com/risingwavelabs/risingwave/network/updates/809611443